app/views/monthlyContributions.scala.html

```diff
@@ -23,6 +23,7 @@
         window.guardian.uatMode = @uatMode;
         window.guardian.user = {
             id: "@user.id",
+            isSignedIn: true,
```

Comment:
I don't think that you need isSignedIn on the `window.guardian.user` object since this object only exists if the user is signed in.  You might need it in the redux store